kernels/portable/cpu/op_constant_pad_nd.cpp

```diff
@@ -189,7 +189,7 @@ Tensor& constant_pad_nd_out(
     CTYPE value_v;
     ET_SWITCH_SCALAR_OBJ_TYPES(
         value_type, ctx, "constant_pad_nd.out", CTYPE_VALUE, [&]() {
-          CTYPE_VALUE val;
+          CTYPE_VALUE val{};
```

Comment:
what is the error message motivating this? it seems silly and reminiscent of misguided linter advice I've seen previously, but perhaps I don't know something. Is it -Wuninitialized? If so, why empty braces instead of `= 0;`?

Reply:
yeah it was that. `COMPLEX_TYPES` is what I was thinking about when not using `=0`. But I guess I can use it when there is no complex.

Reply:
Done.